### PR TITLE
[Finishes #105786170] more basic coverage of applab APIs from mochaTest

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Gets us to the point where (I think) we have minimal coverage of every block in the toolbox.

### CODE REVIEW

File: apps/test/solutions/applab/ec_simple.js [hunk 1]
```diff
@@ -26,6 +26,103 @@ module.exports = {
       },
     },
 
+    // TODO - Data blocks
+
+    // These exercise all of the blocks in Control category
+    // It does not validate that they behave correctly, just that we don't end
+    // up with an errors
```
Comment (davidsbailey) [inline apps/test/solutions/applab/ec_simple.js]: ~~an~~

Comment (Bjvanminnen) [inline apps/test/solutions/applab/ec_simple.js]: also, should address and/or remove the TODO

File: apps/test/solutions/applab/ec_simple.js [hunk 2]
```diff
@@ -26,6 +26,103 @@ module.exports = {
...
+    },
+
+    // These exercise all of the blocks in Control category
+    // It does not validate that they behave correctly, just that we don't end
+    // up with an errors
+    {
+      description: "Variables",
+      editCode: true,
+      xml:
+      // prompt and promptNum are covered in dropletUtilsTest
```
Comment (davidsbailey) [inline apps/test/solutions/applab/ec_simple.js]: nit: indent comment +2

**Comment by davidsbailey** [comment]
LGTM

File: apps/test/solutions/applab/ec_simple.js (hunk 1 above)
Comment (davidsbailey) [inline apps/test/solutions/applab/ec_simple.js]: I'd be interested to know what happens when you run data blocks in mocha tests.

Comment (Bjvanminnen) [inline apps/test/solutions/applab/ec_simple.js]: I think this might be why I just put a TODO here to start rather than just doing the work.

**Pull request merged**